integrations/optimizely/lib/index.js

```diff
@@ -23,8 +23,7 @@ var Optimizely = (module.exports = integration('Optimizely')
   .option('listen', true) // send data via `.track()`
   .option('nonInteraction', false)
   .option('sendRevenueOnlyForOrderCompleted', true)
-  .option('customExperimentProperties', {})
-  .option('customCampaignProperties', {}));
+);
```

Comment:
nit: `var each = require('@ndhoule/each');` not used anymore, can remove

Reply:
Good call, but I think I just brought the usage back! Sorry!